presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/aggregation/GeometryStateSerializerTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.geospatial.aggregation;
+
+import com.esri.core.geometry.ogc.OGCGeometry;
+import com.facebook.presto.operator.aggregation.state.StateCompiler;
+import com.facebook.presto.plugin.geospatial.GeometryType;
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.BlockBuilder;
+import com.facebook.presto.spi.function.AccumulatorStateFactory;
+import com.facebook.presto.spi.function.AccumulatorStateSerializer;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class GeometryStateSerializerTest
+{
+    @Test
+    public void testSerializeDeserialize()
+    {
+        AccumulatorStateFactory<GeometryState> factory = StateCompiler.generateStateFactory(GeometryState.class);
+        AccumulatorStateSerializer<GeometryState> serializer = StateCompiler.generateStateSerializer(GeometryState.class);
+        GeometryState state = factory.createSingleState();
+
+        state.setGeometry(OGCGeometry.fromText("POINT (1 2)"));
+
+        BlockBuilder builder = GeometryType.GEOMETRY.createBlockBuilder(null, 1);
+        serializer.serialize(state, builder);
+        Block block = builder.build();
+
+        assertEquals(GeometryType.GEOMETRY.getObjectValue(null, block, 0), "POINT (1 2)");
+
+        state.setGeometry(null);
+        serializer.deserialize(block, 0, state);
+
+        assertEquals("POINT (1 2)", state.getGeometry().asText());
```

Comment:
swap the arguments to match `assertEquals(expected, actual`

`assertEquals(state.getGeometry().asText(), "POINT (1 2)");`